src/Vatras/Lang/VT.agda

```diff
@@ -0,0 +1,69 @@
+open import Vatras.Framework.Definitions
+module Vatras.Lang.VT (F : 𝔽) where
+
+open import Data.Bool using (if_then_else_)
+open import Data.List using (List; []; _∷_; _++_)
+
+open import Vatras.Data.Prop
+open import Vatras.Framework.Variants using (Rose; Forest; _-<_>-)
+open import Vatras.Framework.VariabilityLanguage
+
+Configuration : Set
+Configuration = Assignment F
+
+data UnrootedVT : 𝔼 where
+  _-<_>- : ∀ {A}
+    → (a : atoms A)
+    → (l : List (UnrootedVT A))
+    → UnrootedVT A
+
+  if[_]then[_] : ∀ {A}
+    → (p : Prop F)
+    → (l : List (UnrootedVT A))
+    → UnrootedVT A
+
+  if[_]then[_]else[_] : ∀ {A}
+    → (p : Prop F)
+    → (l : List (UnrootedVT A))
+    → (r : List (UnrootedVT A))
+    → UnrootedVT A
+
+data VT : 𝔼 where
+  if-true[_] : ∀ {A} → (l : List (UnrootedVT A)) → VT A
+
+vt-leaf : ∀ {A} → atoms A → UnrootedVT A
+vt-leaf a = a -< [] >-
+
+mutual
+  {-|
+  Corresponds to ⟦_⟧* from the dissertation of Paul Bittner.
+  To prove termination and to simplify proofs, this definition is an inlined variant of
+    configure-all c ts = concatMap (configure c) ts
+  -}
+  configure-all : ∀ {A} → Configuration → List (UnrootedVT A) → Forest A
```

Comment:
To me, the arguments seem backwards. As far as I remember, we always put the configuration argument of a semantic last. Is there a reason you chose this order?